Add tentative introduction blog post (#141)
yanthomasdev/lunaria: docs/src/content/docs/blog/introducing-lunaria.md

Maintaining a project for an international audience is no easy task! It involves a lot of effort from both contributors and maintainers to build a sustainable workflow for content to be progressively and consistently localized. Lunaria helps your project by making this effortless -- within a few minutes, you can get a fully tracked project with a dashboard on top that you can share with your contributors to rapidly get into action, all while keeping changes in sync thanks to Lunaria's custom tracking system powered by Git.

And that's not all! Lunaria is **free and open-source software** (FOSS). No sign-ups are needed, and no giving up ownership of your content to another platform. Lunaria's sole goal is -- and will always be -- to help you make the world more inclusive, accessible, and globalized. 

## Current status

Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This release is also an important step to gather more diversified feedback on how Lunaria can mature into a more complete, fast, and stable library in the future.

> As mentioned before, Lunaria is a *toolchain* and comes with different packages meant to help you integrate with, or improve the experience of using Lunaria. Today *also* marks the v0.1 release of the official [Lunaria GitHub Action](https://lunaria.dev/integrations/github-action/) and the [Lunaria integration for Starlight](https://lunaria.dev/integrations/github-action/).

Reviewer: ```suggestion
As mentioned before, Lunaria is a *toolchain* and comes with different packages meant to help you integrate with, or improve the experience of using Lunaria. Today *also* marks the v0.1 release of the official [Lunaria GitHub Action](https://lunaria.dev/integrations/github-action/) and the [Lunaria integration for Starlight](https://lunaria.dev/integrations/starlight/).
```

Should it be this link?

Author: Yes, thanks!

:::

## Why Lunaria?

During the process of making the Astro documentation available for translation, we noticed how hard would be to make a sustainable workflow for both maintainers and contributors. First, most localization management platforms (the ones that help you keep track of and update localizations) have the requirement of owning your content, are paid to some degree, and more importantly, are way too complex for beginners and might not even be available for people in certain countries one might want to reach.

With all this in mind, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built what was then called "Translation Tracker", a tailor-made translation tracking tool that allowed us to work alongside contributors to translate the official Astro documentation in 14 different languages with ease.

Lunaria is the evolution of said tool, with a wider scope, more features, and support for different types of projects -- fun fact, Lunaria was first hinted at in the ["How Astro does i18n" blog post](https://astro.build/blog/astro-i18n/#going-forward)!

## What's in a "toolchain"?

> Lunaria is a *toolchain*, a set of tools that **supercharge your localization workflow** by integrating several parts of your project's infrastructure, including your repository's Git history, your project's file structure, and development platform such as GitHub and GitLab.

Reviewer 2: I _think_ “integrating” needs a preposition? Either “with” or “into”?

Also nit: I guess “or” is more appropriate at the end?

```suggestion
Lunaria is a *toolchain*, a set of tools that **supercharge your localization workflow** by integrating with several parts of your project's infrastructure, including your repository's Git history, your project's file structure, and development platform such as GitHub or GitLab.
```

> As of now -- and that list is going to get even bigger -- the toolchain comes with:

Reviewer 2: Going to suggest switching up the growth point to not interrupt the flow:

```suggestion
As of today, the toolchain includes:
```

- Its own **custom tracking system** that keeps track of your source and localized content changes powered by Git, including a runtime API you can use to build upon. 
- A **complete and easy-to-read status dashboard** you can share with contributors and maintainers.  
- **Official integrations**, such as a [GitHub Action](https://lunaria.dev/integrations/github-action/) and dashboard integration with the [Starlight theme for Astro](https://lunaria.dev/integrations/starlight/).

> 

Reviewer 2: And then maybe follow up _after_ the list with something like this?
```suggestion

And this list is going to grow soon as we add additional tools to complement our core features.

```

## Current status

> Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production, as well as is a way to ask for further feedback and use cases from the community so that Lunaria can mature into an even better project.

Reviewer 2: Not really changing the meaning here, but “as well as is a” is a bit of a mouthful, so I think this lets it run a bit more smoothly:

```suggestion
Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also our chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.
```

Author: Not sure about using "our" here, since I am the only author / actual maintainer of the project, I was purposefully avoiding words such as "we" or "us", but I do like the other changes!

```suggestion
Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also a chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.
```

Reviewer 2: Definitely! Unintentional really — probably my brain unconsciously uncomfortable to say “I”.

Additionally, the officially supported [Lunaria GitHub Action](https://lunaria.dev/integrations/github-action/) and the [Lunaria integration for Starlight](https://lunaria.dev/integrations/starlight/) are also receiving their own v0.1 releases as of today.

> :::caution

Reviewer 2: Small detail, but I wonder if **“Caution”** is the best wording here. Maybe a more positive framing could be something like this?

```suggestion
:::caution[Beta Software]
```

Author: Fair! I'll point out that I generally keep those aside titles in sentence case, not sure which one is more ideal?
```suggestion
:::caution[Beta software]
```

Reviewer 2: That’s fine too, yeah!

Missing features, bugs, and breaking changes are still expected in this early stage across all officially maintained packages. If you find anything wrong or unexpected, please [open an issue on GitHub](https://github.com/yanthomasdev/lunaria/issues/new/choose)!
:::

## Onward

Lunaria is -- and will continue to be -- **free and open-source software** (FOSS). No sign-ups are needed, and no giving up ownership of your content to another platform. Lunaria's sole goal is to help you make the world more inclusive, accessible, and globalized. 

> In response to feedback given, Lunaria will receive constant updates with bug fixes, miscellaneous quality improvements, and brand-new features. At the same time, a lot of brainstorming will happen in the background (and later publicly) on how Lunaria can shape up to be even better in the future -- and believe me, there are lots of exciting ideas being cooked!

Reviewer 2: I guess people don’t _need_ the foreground/background hokey-cokey here? Also suggesting removing the “miscellaneous” as it’s quite vague/weak, almost making the improvements sound not super important.

I _think_ you need “cooked **up**”, not just “cooked” in this context. Sarah will tell us if I’m wrong.

```suggestion
In response to your feedback, Lunaria will receive constant updates with bug fixes, improvements, and brand-new features. At the same time, we will be brainstorming how Lunaria can shape up to be even better in the future -- and believe me, there are lots of exciting ideas being cooked up!
```

Author: I like it! I'll point out again the use of `we`, maybe here is more plausible since "we" might mean me + community in this sense, but I'll let Sarah say what makes more sense!

Reviewer 2: Could even make it more of an invitation to the community maybe to avoid the “we”?

```suggestion
In response to your feedback, Lunaria will receive constant updates with bug fixes, improvements, and brand-new features. Join the community in brainstorming how Lunaria can shape up to be even better in the future -- and believe me, there are lots of exciting ideas being cooked up!
```

Author: LGTM!

## Support

> As usual, maintaining an open-source project is a lot of work, especially when you're an unemployed and broke college student from Latin America. Fortunately, I've been lucky to have received support from [Astro](https://astro.build/) to dedicate some of my time to Lunaria these past months and bring it to its current state.

Reviewer 2: I think you can skip “as usual” and it feels even more direct:

```suggestion
Maintaining an open-source project is a lot of work, especially when you're an unemployed and broke college student from Latin America. Fortunately, I've been lucky to have received support from [Astro](https://astro.build/) to dedicate some of my time to Lunaria these past months and bring it to its current state.
```

> If your company or project uses Lunaria, supporting it through [GitHub Sponsors](https://github.com/sponsors/yanthomasdev) would go a long way into helping it be maintained in the foreseeable future, and even better, allow me to live sustainably working full-time on open-source.

Reviewer 2: Suggesting some small language tweaks here for flow:

```suggestion
If your company or project uses Lunaria, supporting development through [GitHub Sponsors](https://github.com/sponsors/yanthomasdev) will help guarantee its long-term maintenance, and even better, allow me to live sustainably while working full-time on open-source.
```

Lunaria is a complete **localization management toolchain** built especially for open-source projects. 

Lunaria is already being used in production to help track over 5,000 content files in over a dozen locales in big projects such as [Astro](https://i18n.docs.astro.build/), [Starlight](https://i18n.starlight.astro.build/), [SolidJS](https://docs.solidjs.com/i18n-status/), and [VitePress](https://vitepress.dev/_translations/).

:::tip[Ready to get started?]
Read the official ["Getting Started" guide](https://lunaria.dev/getting-started/) to learn more about and set up Lunaria in your project. 
:::

## Why Lunaria?

While making the Astro documentation available for translation, we noticed how hard it was to build a sustainable workflow for both maintainers and contributors. Most localization management platforms (the ones that help you keep track of and update localizations) have the requirement of owning your content and are paid to some degree. More importantly, these platforms are way too complex for beginners or might not even be available for people in certain countries one might want to reach.

> With all this in mind, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built a tailor-made translation tracking tool that allowed us to work with contributors to translate the official Astro documentation into 14 different languages with ease.

Author: Similar to my comment before, weird to say "our" here since Lunaria != Astro and this isn't a blog post on the Astro blog, I guess changing a bit fixes this?

```suggestion
Wanting a truly free, open-source solution that integrated seamlessly into the translators' and maintainers' normal workflow, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built a tailor-made translation tracking tool. The community contributors used this to organize and mobilize, translating the official Astro documentation into 14 different languages with ease.
```

> Lunaria is the evolution of that bespoke tool, with a wider scope, more features, and support for different types of projects -- fun fact, Lunaria was first hinted at in the ["How Astro does i18n" blog post](https://astro.build/blog/astro-i18n/#going-forward)!

Author: Thanks! Not sure if you're aware but the `--` is converted to emdashes by default with the default set of plugins for Markdown in Astro

> Lunaria is the evolution of that bespoke tool, with a wider scope, more features, and support for different types of projects -- fun fact, Lunaria was first hinted at in the ["How Astro does i18n" blog post](https://astro.build/blog/astro-i18n/#going-forward)!

Reviewer 3: Clearly I am not 😅

## What's in a "toolchain"?

Lunaria is a *toolchain*, a set of tools that **supercharge your localization workflow** by integrating with several parts of your project's infrastructure, including your repository's Git history, your project's file structure, and development platform such as GitHub or GitLab.

As of today, the toolchain includes:

- Its own **custom tracking system** that keeps track of your source and localized content changes powered by Git, including a runtime API you can use to build upon. 
- A **complete and easy-to-read status dashboard** you can share with contributors and maintainers.  
- **Official integrations**, such as a [GitHub Action](https://lunaria.dev/integrations/github-action/) and dashboard integration with the [Starlight theme for Astro](https://lunaria.dev/integrations/starlight/).

And this list is going to grow soon as we add additional tools to complement our core features.

## Current status

> Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also a chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.

Author: Not sure about using "we" in this blog post? There's no bigger organization behind it, so it feels weird to talk in the plural, also because the last few paragraphs very directly talk about me (the author) in singular.  I guess we are just too used to using it because of the Astro blog posts 😆

> Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also a chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.

Reviewer 3: Right, I had left this comment then saw you used "I" later. Totally find to make it an I here too!

> Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also a chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.

Author: I guess this works?
```suggestion
Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. I also encourage feedback and would love to learn about *your* use cases so that Lunaria can mature into an even better project.
```

> Today marks the release of Lunaria v0.1. This minor release means that Lunaria is ready to be tested and used in production. This is also a chance to ask for feedback and learn about new use cases from the community so that Lunaria can mature into an even better project.

Reviewer 3: *I* think it works! Use whatever you're comfortable with!
